Update availability-replica-is-disconnected.md (#9770)
MicrosoftDocs/sql-docs: docs/database-engine/availability-groups/windows/availability-replica-is-disconnected.md

   Use the following commands to test connectivity in both directions from Node1 to Node2 and Node2 to Node1:

   ```PowerShell

>    $computer = "remote_node" 	 # Replace with node name in your environment.

Reviewer: ```suggestion
   $computer = "remote_node" 	  # Replace with node name in your environment.
```

>    $computer = "remote_node" 	# Replace with Node name as per your environment.

Reviewer: ```suggestion
   $computer = "remote_node" 	 # Replace with node name in your environment.
```

   1. Then you can test the connection to the remote node. For example:

      ```PowerShell

>       $computer = "remote_node" # Replace with Node name as per your environment.

Reviewer: ```suggestion
      $computer = "remote_node" # Replace with node name in your environment.
```
